pandas/core/frame.py

```diff
@@ -6534,7 +6534,7 @@ def dropna(
     @overload
     def drop_duplicates(
         self,
-        subset: Hashable | Sequence[Hashable] | None = ...,
+        subset: Hashable | Sequence[Hashable] | set | None = ...,
```

Comment:
I think you should just replace `Sequence[Hashable]` with `Iterable[Hashable]` here (and in the other overloads), because even a `dict` works now.

Also, I suggest making a separate PR in `pandas-stubs` once this PR is accepted.

Reply:
@Dr-Irv,

Updated. Could you specify which docs I should be updating to reflect these changes?

Also, what is the difference between `pandas` and `pandas-stubs`?
I am a new contributor and am happy to raise the PR there but would like to understand more about each repo.

Reply:
> Also, what is the difference between `pandas` and `pandas-stubs`? I am a new contributor and am happy to raise the PR there but would like to understand more about each repo.

`pandas` is the code that executes.

`pandas-stubs` is a set of typing declarations meant for users to use when type checking their code.  It is bundled within Visual Studio Code.   See https://github.com/pandas-dev/pandas-stubs/

Reply:
Thanks for clarifying. I will do as you suggested and open a PR there once this one is merged.